### PR TITLE
[KC-1] - Redux webcam

### PR DESCRIPTION
**Pull request by JohnStuartBarrie**
*** Description ***
Redux webcam selector

### CODE REVIEW

File: src/view/kc-display.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { LitElement, html } from 'lit-element'
+
+export class KCDisplay extends LitElement {
+  render () {
+    return html`
+    <div>
+      <kc-webcam-selector></kc-webcam-selector>
```
Comment (dquilter) [inline src/view/kc-display.js:7]: Do you not have to import these? 

I'm not saying this is wrong, just want to understand what's happening...

File: src/view/kc-webcam-selector.js [hunk 2]
```diff
@@ -1,19 +1,34 @@
+import { connect } from 'pwa-helpers'
 import { LitElement, html } from 'lit-element'
+import { store } from '../model/store.js'
+import { changeWebcam } from '../action/change-webcam.js'
+
+export class WebCamSelector extends connect(store)(LitElement) {
+  static get properties () {
+    return {
+      webCameDevices: { type: Array }
```
Comment (dquilter) [inline src/view/kc-webcam-selector.js]: This property name is freaking me out.

File: src/view/kc-display.js (hunk 1 above)
Comment (JohnStuartBarrie) [reply src/view/kc-display.js:7]: These are declared at the browser level. They can be dynamically imported (you can see an example of the dynamic definition in spikes). in this case, they are defined in the html at the beginning in the same way. They become like normal tags like div. I think it is this aspect that I find so agreeable about this technic... for the moment.

File: src/view/kc-webcam-selector.js (hunk 2 above)
Comment (JohnStuartBarrie) [reply src/view/kc-webcam-selector.js]: fixed

**Review by dquilter — APPROVED** [review]

**Pull request merged**